src/main/java/az/gdg/msteam/service/impl/MemberServiceImpl.java

```diff
@@ -0,0 +1,130 @@
+package az.gdg.msteam.service.impl;
+
+import az.gdg.msteam.exception.MemberNotFoundException;
+import az.gdg.msteam.mapper.MemberMapper;
+import az.gdg.msteam.model.MemberResponse;
+import az.gdg.msteam.model.dto.MemberDto;
+import az.gdg.msteam.model.entity.Member;
+import az.gdg.msteam.repository.MemberRepository;
+import az.gdg.msteam.service.MemberService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class MemberServiceImpl implements MemberService {
+
+    MemberRepository memberRepository;
+    private boolean isValid;
+
+    public MemberServiceImpl(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    @Override
+    public List<MemberDto> getAllMembers() {
+        List<Member> members = new ArrayList<>();
+
+        for (Member member : memberRepository.findAll()) {
+            members.add(member);
+        }
+
+        return MemberMapper.INSTANCE.entityToDtoList(members);
+    }
+
+    @Override
+    public String createMember(MemberDto memberDto) {
+        isValid = true;
+        String message = isMemberValid(memberDto);
+        if (isValid && findByEmail(memberDto.getEmail()) == null) {
+            memberRepository.save(MemberMapper.INSTANCE.dtoToEntity(memberDto));
+            message = "Member is created";
+        } else {
+            message = "Email already exist";
+        }
+        return message;
+    }
+
+    @Override
+    public Member findByEmail(String email) {
+        return memberRepository.findByEmail(email);
+    }
+
+    @Override
+    public String deleteMember(Integer id) {
+        MemberResponse response = new MemberResponse();
+        if (findById(id) != null) {
+            memberRepository.deleteById(id);
+            response.setMessage("Member deleted");
+        } else {
+            response.setMessage("Id is not available");
+        }
+        return response.getMessage();
+    }
+
+    @Override
+    public Member findById(Integer id) {
+        Member member = null;
+        try {
+            member = memberRepository.findById(id).orElseThrow(() -> new MemberNotFoundException("Member doesn't exist with this id"));
```

Comment:
add id to exception, so it is easier for you to understand the problem from logs. You did not add any logs, why ?